mcodingbot/plugins/highlights.py

```diff
@@ -2,21 +2,45 @@
 
 import asyncio
 from collections import defaultdict
-from typing import NamedTuple
+from typing import TYPE_CHECKING, NamedTuple, cast
 
 import crescent
 import hikari
 from asyncpg import UniqueViolationError
 from floodgate import FixedMapping
+from toolbox.members import calculate_permissions
```

Comment:
should be `import toolbox` or `from toolbox import calculate_permissions`. I prefer the first one OFC.